packages/buttons/package.json

```diff
@@ -19,7 +19,7 @@
     "start": "../../utils/scripts/start.sh"
   },
   "dependencies": {
-    "@zendeskgarden/container-buttongroup": "^0.1.11",
+    "@zendeskgarden/container-buttongroup": "^0.2.0",
     "@zendeskgarden/css-variables": "^6.1.1",
```

Comment:
Shouldn't this be a locked version in `devDependencies`?

Reply:
It looks like this isn't even being used as a devDependency. Removing.